src/GRA.Web/Views/Avatar/Share.cshtml

```diff
@@ -1,5 +1,7 @@
 ﻿@model GRA.Controllers.ViewModel.Avatar.ShareViewModel
 
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
SharedLocalizer doesn't need to be injected here as it's coming in through the _ViewImports.

Reply:
Fixed in commit a644d762f46ed05c11f0f643e52b135003a8f354